css/index.css

```diff
@@ -1,3 +1,340 @@
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+	font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+body.container {
+	width: 1024px;
+	margin: 0 auto;
+}
+
+.nav{
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+    justify-content: space-between;
+    align-items: center;
+	margin-top: 25px;
+}
+
+nav {
+	display: flex;
+	flex-direction: row;
+	flex-wrap:wrap;
+	justify-content: space-between;
+	width: 700px;
+}
+
+nav a{
+    text-decoration: none;
+	color: darkgray;
+}
+
+@media(max-width: 800px) {
+	.nav{
+		display:flex;
+		flex-direction:column-reverse;
+		flex-wrap: wrap;
+		justify-content: center;
+		width: 800px;
+	}
+
+	.logo{
+		margin-bottom: 20px;
+	}
+}
+
+@media(max-width:500px){	
+	.nav{
+		display: flex;
+		flex-direction: column-reverse;
+		flex-wrap: wrap;
+		width: 500px;
+	}
+
+	nav{
+		display: flex;
+		flex-direction: column;
+		flex-wrap: wrap;
+		align-items:center;
+		font-size: 30px;
+		width: 500px;
+	}
+
+	.logo{
+		padding: 0 170px 15px 200px;
+		border-bottom: 1px solid black;
+	}
+
+	.navline{
+		width:500px;
+		text-align:center;
+		border-bottom: 1px solid black;
+	}
+}
+
+.main{
+	display: flex;
+	flex-direction: row;
+	flex-wrap: wrap;
+	justify-content: space-evenly;
+	align-items: center;
+	margin-top: 50px;
+}
+
+@media(max-width: 800px) {
+	.main{
+		display: flex;
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: center;
+		align-items: center;
+		width:800px;
+		margin-top: 50px;
+	}
+	
+	.circlepic{
+		display: none;
+	}
+}
+
+@media(max-width: 500px) {
+	.main {
+		display:flex;
+		flex-wrap: wrap;
+		justify-content:center;
+		align-items: center;
+		width: 500px;
+		margin-top: 30px;
+	}
+
+	.circlepic {
+		display: none;
+	}
+}
+
+
+.title{
+	font-size: 80px;
+	text-align: center;
+}
+
+.getstarted{
+	width: 190px;
+	margin-top: 20px;
+	margin-left: 65px;
+	font-size: 20px;
+	border: 1px solid black;
+	padding: 5px 45px 5px 45px;
+}
+
+.line1{
```

Comment:
I would remove this and use section borders to make the line instead of a separate line